dbschema/seeds/017.partnerships.edgeql

```diff
@@ -0,0 +1,65 @@
+with
+partnershipsJson := to_json($$[
```

Comment:
Just a questions as I have not seen this before.  What is the purpose of the `$$` at the beginning and end of the array?